caddy/examples/config_example.json

```
@@ -0,0 +1,172 @@
+{
+    "admin": {
+        "disabled": true
+    },
+    "logging": {
+      "logs": {
+        "default": {"level":"DEBUG", "encoder": {"format":"console"}}
+      }
+    },
+    "apps": {
+        "http": {
+            "servers": {
+                "metrics": {
+                    "listen": [
+                        ":9091"
+                    ],
+                    "routes": [
+                        {
+                            "match": [
+                                {
+                                    "path": [
+                                        "/metrics"
+                                    ]
+                                }
+                            ],
```

I find that the excessive line breaks hurts readability. Something like this is easier to read:
```suggestion
                    "listen": [":9091"],
                    "routes": [
                        {
                            "match": [{"path": ["/metrics"]}],
```

---

With the new main.go we can import the YAML adapter and use YAML everywhere. I'm going to remove this `json` config. Hopefully that will increase readability.